lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -730,11 +776,13 @@ def main():
         # if we have a provider, edit it
         if provider:
             res_args = manageiq_provider.edit_provider(provider, name, provider_type, endpoints, zone_id, provider_region,
-                                                       host_default_vnc_port_start, host_default_vnc_port_end)
+                                                       host_default_vnc_port_start, host_default_vnc_port_end,
```

Comment:
it get a little out of hand here :-)

can you re-factor this functions to use a dict instead of a long list of args ?

Reply:
@yaacov I can have a go but as you can probably tell I'm no python expert. @cben mentioned he had an idea to refactor this so I was just continuing to add providers... :)

Reply:
the easiest way I can think of:

define some function
```
def some_func(provider_info):
    post(provider_info["name"], provider_info['ems_id'])
    ...
```
call some function
```
provider_info = dict(name = "hello", ems_id = "world")
some_func(provider_info)
```
@cben sounds legit to you ?

Reply:
@snecklifter ^^

Reply:
if pep is ok with it ... LGTM as is